ehrenamt-justiz-online/.env.production

```
@@ -0,0 +1 @@
+VITE_VUE_APP_API_URL=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)